### PR TITLE
sip: added headers to SIP inbound and output create requests

### PR DESCRIPTION
**Pull request by s-hamdananwar**

### CODE REVIEW

**Review by dennwc — APPROVED** [review]
LGTM! Please don't forget to add the changeset.

File: packages/livekit-server-sdk/src/SipClient.ts [hunk 1]
```diff
@@ -166,11 +166,15 @@ export class SipClient extends ServiceBase {
...
    * @param numbers - phone numbers of the trunk
    * @param opts - CreateSipTrunkOptions
+   * @param headers - SIP X-* headers to be included in 200 OK responses (optional)
+   * @param headersToAttributes - map of SIP X-* headers from INVITE to SIP participant attributes (optional)
    */
   async createSipInboundTrunk(
     name: string,
     numbers: string[],
     opts?: CreateSipInboundTrunkOptions,
+    headers?: { [key: string]: string },
```
Comment (davidzhao) [inline packages/livekit-server-sdk/src/SipClient.ts:176]: these should be included in opts

File: packages/livekit-server-sdk/src/SipClient.ts [hunk 2]
```diff
@@ -212,12 +218,16 @@ export class SipClient extends ServiceBase {
...
+   * @param headers - SIP X-* headers to be included in 200 OK responses (optional)
+   * @param headersToAttributes - map of SIP X-* headers from INVITE to SIP participant attributes (optional)
    */
   async createSipOutboundTrunk(
     name: string,
     address: string,
     numbers: string[],
     opts?: CreateSipOutboundTrunkOptions,
+    headers?: { [key: string]: string },
+    headersToAttributes?: { [key: string]: string },
```
Comment (davidzhao) [inline packages/livekit-server-sdk/src/SipClient.ts:230]: same as above.

File: packages/livekit-server-sdk/src/SipClient.ts (hunk 1 above)
Comment (s-hamdananwar) [reply packages/livekit-server-sdk/src/SipClient.ts:176]: oh okay, was debating whether these should be part of options. will update now

**Comment by s-hamdananwar** [comment]
> Please don't forget to add the changeset.

thanks for the reminder, I have now added it!

**Review by davidzhao — APPROVED** [review]

**Pull request merged**